shishito/library/modules/runtime/environment/appium_control_environment.py

```diff
@@ -0,0 +1,70 @@
+from appium import webdriver
+#from selenium import webdriver
+import time
+
+from shishito.library.modules.runtime.environment.shishito_environment import ShishitoEnvironment
+
+
+class ControlEnvironment(ShishitoEnvironment):
+    """ Appium control environment. """
+
+    CAPABILITIES = (
+        ('platformName', 'platformName', True, None),
+        ('platformVersion', 'platformVersion', True, None),
+        ('deviceName', 'deviceName', True, None),
+        ('app', 'app', True, None),
+        ('appiumVersion', 'appiumVersion', True, None),
+    )
+
+    def call_browser(self, combination, capabilities=None):
+        """ Starts browser """
+
+        # get browser capabilities and profile
+        if capabilities:
+            capabilities.update(self.get_capabilities(combination))
+        else:
+            capabilities = self.get_capabilities(combination)
+
+        use_saucelabs = self.shishito_support.gid('use_saucelabs')
+        if use_saucelabs is not None and use_saucelabs.lower() == 'true':
+            saucelabs = self.shishito_support.gid('saucelabs')
```

Comment:
why it has `use_saucelabs` and `saucelabs` arg. Why it is not managed just by saucelabs arg (if not None then use it)

btw personally I would use shorted
`if use_saucelabs and use_saucelabs.lower() == 'true'`

second note is that you have these check quite often in code. Consider adding typed args 
somethign like 
self.shishito_support.gid('use_saucelabs', bool)
which returns you converted argument